DietPi-Software | Kodi: Fix install on RPi Buster via 3rd party repo/builds

MichaIng opened the pull request:
**Status**: Testing
- [x] Fix /DietPi/dietpi/misc/start_kodi for RPi4, since the custom package ships a different binary
- [x] Changelog

**Testing**:
- [ ] Install on RPi4
- [ ] Install on other RPis on Buster
- [x] RPi4, running `start_kodi` from console, respectively via dietpi-autostart

Fixes: #3031

**Commit list/description**:
+ DietPi-Software | Kodi: Fix Kodi install on RPi Buster, since archive.raspbian.org does not yet contain a Buster-compatible package. Use a 3rd-party repo for RPi1-3 for now and a custom compiled package for RPi4, to have full GPU acceleration.

MichaIng committed:
v6.26

+ DietPi-Software | Kodi: Fix Kodi install on RPi Buster, since archive.raspbian.org does not yet contain a Buster-compatible package. Use a 3rd-party repo for RPi1-3 for now and a custom compiled package for RPi4, to have full GPU acceleration.

MichaIng committed:
v6.26

+ DietPi-Software | Kodi: Use RPi4 specific start script. Prefer mawk over sed, since it is faster.

MichaIng committed:
v6.26

+ CHANGELOG | Kodi: Resolved an issue where Kodi failed on RPi Buster and RPi4

MichaIng committed:
v6.26

+ DietPi-Software | Kodi: start_kodi will now run plain kodi binary with --standalone argument. kodi-rpi4 and kodi-standalone are shell wrappers only that do nothing that we want or require.

vinanrra commented:
I had swap files, installed with the new config files and my Kodi its frozen at KODI start with the kodi version.

[Image ](https://kodi.tv/sites/default/files/styles/hero/public/article/field_image/kodi_18_3_leia.jpg?itok=HTHCKIyh) is from internet, but its the same.

MichaIng commented:
@vinanrra 
Did you try to install Kodi on RPi with DietPi OS (via DietPi-Software), using the code this pull request? This is about to test if this install method/code is working properly on all RPi devices with DietPi OS.

vinanrra commented:
Yes, I even did on a fresh install 2 times, because thought I did something wrong

MichaIng committed:
v6.26

+ DietPi-Software | Kodi: Update RPi4 Kodi download link to new v18.4. URL check fails, but download via redirect works.

MichaIng committed:
Merge branch 'dev' into kodi

MichaIng committed:
v6.26

+ DietPi-Software | Kodi: fbset seems to be not required for the custom Buster+RPi4-compatible Kodi 18 builds
